packages/babel/test/fixtures/proposal-decorators/main.js

```diff
@@ -1,6 +1,8 @@
 @annotation
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
nit: maybe it would be better here to disable this rule for the whole file? a decorator interleaved with the comment looks a little bit off, no big deal though

Reply:
Done